department-of-veterans-affairs/va.gov-cms
# VAGOV-3911 migrate careers

To test:
1. Go to /admin/structure/migrate/manage/va_new_benefits/migrations/va_benefits_careers/execute and run the migration.
2. Confirm that there are now 15 benefits detail pages in draft with Owner: `Careers and employment benefits hub`
3. Edit one and make sure that it has a menu entry.
